### PR TITLE
fix: allow to deep-require next-auth

### PR DESCRIPTION
**Pull request by masone**

### CODE REVIEW

File: next.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ module.exports = {
           "**/configuration/**/*",
           "**/configuration/*",
           "next/*",
+          "next-auth/*"
```
Comment (GutierrezGo) [inline next.js:31]: you just forgot the `,`

**Review by GutierrezGo — APPROVED** [review]

**Review by Averethel — APPROVED** [review]
Normally we first add new rules in affected projects and move them here afterwards.
That being said - if you're confident about next-auth go for it after adding that missing comma :)

File: next.js (hunk 1 above)
Comment (masone) [reply next.js:31]: 🙈

Comment (masone) [reply next.js:31]:
```suggestion
          "next-auth/*",
```

**Comment by masone** [comment]
closing to free some memory on the dev cluster, please keep the branch so we can build on it later

**Review by lkappeler — APPROVED** [review]
Nice :)

**Comment by masone** [comment]
we decided to roll with next-auth. let's also roll with this then :)

**Pull request merged**